smart-send-logistics/includes/frontend/class-ss-shipping-frontend.php

```diff
@@ -63,56 +88,56 @@ public function display_ss_pickup_points($method, $index)
 
             $meta_data = $method->get_meta_data();
 
-            if ($chosen_shipping &&
+            if (
+                $chosen_shipping &&
                 ($method_id == 'smart_send_shipping') &&
                 ($chosen_shipping == $shipping_id) &&
-                (stripos($meta_data['smart_send_shipping_method'], 'agent') !== false)) {
+                (stripos($meta_data['smart_send_shipping_method'], 'agent') !== false)
```

Comment:
Abstract this check into some private functions for `isSmartSendShippingMethod($id): bool` and `isPickupPointMethod(string $name): bool`